hack/scripts/provision.sh

```diff
@@ -294,7 +294,7 @@ install_flintlockd() {
 	say "Installing flintlockd version $tag to $INSTALL_PATH"
 
 	if [[ "$tag" == "$DEFAULT_VERSION" ]]; then
-		tag=$(latest_pre_release_tag "$FLINTLOCK_REPO")
+		tag=$(latest_release_tag "$FLINTLOCK_REPO")
```

Comment:
Do we need the `latest_pre_release_tag` function after this patch?

Reply:
technically no. i know i am usually ruthless when it comes to deleting dead code, but for some reason I had great attachment to this one 🤣

Reply:
it's dead i killed it hope you are happy 😭 😭

Reply:
:scream: